src/main/resources/views/CommercialVehicles/Abandoned.hbs

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html>
+  <head>
+    <title id="documentType">{{documentType}}</title>
+    <style type="text/css" media="print, screen">
+        {{~> assets/stylesheets/version-1 this}}
+        {{~> assets/stylesheets/cvs-version-1 this}}
+        {{~> assets/stylesheets/Abandoned this}}
+    </style>
+  </head>
+  <body>
+    <!-- certificate -->
+    <div class="front-page">
+      <div class="certificate">
+        <div class="certificate__wrapper">
+
+          <!-- header -->
+          <div class="header">
+            <img src="{{root}}/assets/images/dvsa_crest.png" alt="DVSA" class="header__crest-img"/>
+            <div class="title-text">
+              <div class="bold-text" id="firstPageTitle">
+                Notification of Failure to Comply with the Conditions of Acceptance of a
+                  {{titleTextIncludingRollingHeaders}} ({{documentType}})
+              </div>
+            </div>
+            <div class="standard-text regulation-text" id="regulationText">
+              {{regulationText}}
+            </div>
+          </div>
+
+          <!-- footer -->
+          <div class="footer-left">
+            <b id="footerDocumentType">{{documentType}}</b>
+            <span id="footerTestNumber"><b> (DVSA{{formNumber}})</b></span>
```

Comment:
On ticket example, only documentType is bold

Reply:
Mel said to follow IVA/MSVA footer. that is why it is bold on the left hand side of the footer and the size of it as well